canonical-web-and-design/manager.assets.ubuntu.com
# 404 page

PR #7 again:
- error_404 view
- handler404 = error_404
- Add hex image for the view
- Create 404 template with just the hex image
- Create new layout without the header
- Position image in the middle of the page
## QA
- Disable DEBUG in `assets_manager/dev-settings.py`
- Run the server locally
- Visit a non-existent page, check 404 image shows
